src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -220,6 +225,20 @@ private void validateTable(TableModel table, Errors errors, SchemaValidationCont
         }
     }
 
+    private void validateDataTypes(List<ColumnModel> columns, Errors errors) {
+        List validDataTypes = new ArrayList(Arrays.asList(TableDatatypes.values()));
```

Comment:
In Java you should use type parameters when using generic (parameterized) types. They have no runtime effect but allow the compiler to do better type checking.
```suggestion
        List<DataTypes> validDataTypes = new ArrayList<>(Arrays.asList(TableDatatypes.values()));
```
Also I don't think `new ArrayList()` is needed here
```
        List<DataTypes> validDataTypes = Arrays.asList(TableDatatypes.values());
```